src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
I'm surprised that there is UTF-8, I never encountered a part of windows using it I think.

For Auto / Unicode yes it's the same, only specifying nothing is problematic. It seem that `Unicode` is more used in the lib currently (I could find only one use of `Auto` but GitHub search isn't really good) so yes let's keep using it.